src/nodes/array/splice.js

```diff
@@ -0,0 +1,59 @@
+const dayjs = require('dayjs')
+const AbstractNode = require('../abstract')
+
+class ArraySplice extends AbstractNode {
+
+  static metadata() {
+    return {
+      name: 'Splice',
+      code: 'array/splice',
+      type: 'modifier',
```

Comment:
change values of inputs, so  should be callable and has execution flow input/output